docs/tutorials/services/source-ip.md

```diff
@@ -132,7 +132,7 @@ client_address=10.240.0.5
 client_address=10.240.0.3
 ```
 
-Note that these are not your IPs, they're cluster internal IPs. This is what happens:
+Note that these are not right client IPs, they're cluster internal IPs. This is what happens:
```

Comment:
What do you mean by "right client IPs"? In this context, does "right" mean "correct"?